Formula/y/yank.rb

```diff
@@ -26,7 +26,7 @@ def install
   end
 
   test do
-    (testpath/"test.exp").write <<~EOS
+    (testpath/"test.exp").write <<~EXPECT
```

Comment:
any refs for this, EXPECT does not do syntax highlighting for me.

Reply:
Could use TCL instead as mainly an extension on top of that.

Reply:
is there any issue on ruby-lsp for these languages support?

Reply:
I think it is mainly a PR situation.

https://github.com/Shopify/ruby-lsp/blob/main/vscode/grammars/ruby.cson.json shows support for HTML, HAML, XML, SQL, GQL/GRAPHQL, CSS, CPP, C, JS/JAVASCRIPT, JQUERY, SH/SHELL, LUA, RUBY, YAML/YML, SLIM

Reply:
make sense 👍